app/src/playStore/java/com/eventyay/organizer/core/event/create/UpdateEventFragment.java

```diff
@@ -304,38 +297,49 @@ public void close() {
 
     private void setupPlacesAutocomplete() {
 
-        ApplicationInfo ai = null;
+        ApplicationInfo applicationInfo = null;
         try {
-            ai = getContext().getPackageManager().getApplicationInfo(getContext().getPackageName(), PackageManager.GET_META_DATA);
+            applicationInfo = getContext().getPackageManager().getApplicationInfo(getContext().getPackageName(), PackageManager.GET_META_DATA);
         } catch (PackageManager.NameNotFoundException e) {
-            e.printStackTrace();
+            Timber.e(e);
         }
-        Bundle bundle = ai.metaData;
-        String placesApiKey = bundle.getString("com.google.android.geo.API_KEY");
+        Bundle bundle = applicationInfo.metaData;
 
-        Places.initialize(getActivity().getApplicationContext(), placesApiKey);
+        String mapboxAccessToken = bundle.getString(getString(R.string.mapbox_access_token));
 
-        AutocompleteSupportFragment autocompleteFragment = (AutocompleteSupportFragment)
-            getChildFragmentManager().findFragmentById(R.id.autocomplete_fragment);
+        binding.form.selectLocationButton.setOnClickListener(view -> {
 
-        autocompleteFragment.setPlaceFields(Arrays.asList(Place.Field.ID, Place.Field.NAME, Place.Field.LAT_LNG, Place.Field.ADDRESS));
-
-        autocompleteFragment.setOnPlaceSelectedListener(new PlaceSelectionListener() {
-            @Override
-            public void onPlaceSelected(Place place) {
-
-                Timber.d(place.getAddress());
-                Event event = binding.getEvent();
-                event.latitude = place.getLatLng().latitude;
-                event.longitude = place.getLatLng().longitude;
-                event.locationName = place.getAddress();
-                event.searchableLocationName = place.getName();
+            if (mapboxAccessToken.equals("YOUR_ACCESS_TOKEN")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/fossasia/open-event-orga-app/pullRequest?prid=4479632)